### PR TITLE
EHPR-146 | Setup dev/test vanity url

### PR DESCRIPTION
**Pull request by fwpushan**
- Create SSL certificate on respective aws workspace account
- Copy associated CNAME record/value from certificate and create entry on *.freshoworks.club hosted zone 
- Create CNAME entry with respective domain name in *.freshoworks.club hosted zone.

### CODE REVIEW

File: .github/workflows/deploy-prod.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ env:
 jobs:
   deploy:
     runs-on: ubuntu-latest
-    timeout-minutes: 10
+    timeout-minutes: 20
```
Comment (arranfw) [inline .github/workflows/deploy-prod.yml:13]: This issue originated from TF didn't it? Increasing to 20 minutes didn't help us and will probably just waste more time in the future, if deploys are taking more than 10 minutes they're probably hanging right?

**Review by arranfw — CHANGES_REQUESTED** [review]
Do you have more details for what other work was needed to achieve this? I assume there were some FW AWS changes?

One comment

File: Makefile [hunk 2]
```diff
@@ -33,13 +34,25 @@ APP_SRC_BUCKET = $(NAMESPACE)-app-dist
 TERRAFORM_DIR = terraform
 export BOOTSTRAP_ENV=terraform/bootstrap
 
+ifeq ($(ENV_NAME), prod)
+DOMAIN=
```
Comment (dbayly-freshworks) [inline Makefile:38]: Is there any risk for having this variable blank on prod

Comment (fwpushan) [reply Makefile:38]: nope, blank value configures terraform and no dns + ssl association will be created.

File: .github/workflows/deploy-prod.yml (hunk 1 above)
Comment (fwpushan) [reply .github/workflows/deploy-prod.yml:13]: CF update is taking for time 5-6m. Previously we executed CF update from local machine ....

**Review by arranfw — APPROVED** [review]

**Review by dbayly-freshworks — APPROVED** [review]

**Pull request merged**